app/presenters/vendor_api/single_application_presenter.rb

```diff
@@ -95,6 +95,11 @@ def get_rejection
           reason: application_choice.offer_withdrawal_reason,
           date: application_choice.offer_withdrawn_at.iso8601,
         }
+      elsif application_choice.rejected_by_default?
```

Comment:
I wonder if there are any older applications without rejection reasons and which aren't RBD?

Reply:
great ⭐